win_build/vcpkg_config_msbuild_x64/vcpkg.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "boinc-msbuild-x64",
+    "version-string": "7.19.0",
+    "dependencies":
+    [
+        {
+          "name": "curl",
+          "features": ["schannel"],
+          "default-features": false
+        },
+        "openssl",
+        "rappture",
+        "wxwidgets",
+        "gtest",
+        "ftgl",
+        "opencl"
+    ]
+}
+
```

Comment:
Please remove two extra lines at the end

Reply:
done.